app/models/like.rb

```diff
@@ -2,10 +2,12 @@ class Like < ApplicationRecord
     belongs_to :user
     belongs_to :post
 
-    after_create :update_like_counter
-  
+    after_save :update_like_counter
+
+    private
+   
     def update_like_counter
-      post.update(likes_counter: post.likes.count)
-    end
+        post.update(likes_counter: post.likes.count)
+      end
   end
```

Comment:
It would be nice if you make test for this method to make your project follow the given requirements 👍